include/deal.II/base/work_stream.h

```diff
@@ -747,36 +747,59 @@ namespace WorkStream
 
         tf::Task last_copier;
 
+        struct CopyChunk
+        {
+          std::vector<CopyData> copy_datas;
+
+          CopyChunk(const unsigned int chunk_size,
+                    const CopyData    &sample_copy_data)
+            : copy_datas(chunk_size, sample_copy_data)
+          {}
+        };
+
         // idx is used to connect each worker to its copier as communication
         // between tasks is not supported. It does this by providing a unique
         // index in the vector of pointers copy_datas at which the copy data
         // object where the work done by work task #idx is stored
         unsigned int idx = 0;
 
-        std::vector<std::unique_ptr<CopyData>> copy_datas;
+        std::vector<std::unique_ptr<CopyChunk>> copy_chunks;
```

Comment:
Can you document this? Something like:
Store a handle to a CopyChunk for each chunk. The actual data will be allocated in the worker and freed in the copier.